### PR TITLE
update config sample files

### PR DESCRIPTION
**Pull request by mmattel**
This PR updates the config sample files based on the recently merged core https://github.com/owncloud/core/pull/36325

Backport to 10.3 necessary

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc [hunk 1]
```diff
@@ -346,12 +337,12 @@ settings which are further used for app or client authentication. Browser logon
...
 ....
 
-=== Allows to specify additional login buttons on the logon screen for e.g. SSO integration.
-
-'login.alternatives' => [
-  ['href' => 'https://www.testshib.org/Shibboleth.sso/ProtectNetwork?target=https%3A%2F%2Fmy.owncloud.tld%2Flogin%2Fsso-saml%2F', 'name' => 'ProtectNetwork', 'img' => '/img/PN_sign-in.gif'],
-  ['href' => 'https://www.testshib.org/Shibboleth.sso/OpenIdP.org?target=https%3A%2F%2Fmy.owncloud.tld%2Flogin%2Fsso-saml%2F', 'name' => 'OpenIdP.org', 'img' => '/img/openidp.png'],
-]
+=== Defines additional login buttons on the logon screen
+Allows to specify additional login buttons on the logon screen for e.g. SSO integration
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc]: The start of this sentence doesn't read correctly. I suggest starting with: "Allows for specifying additional..."

File: modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc [hunk 2]
```diff
@@ -405,11 +396,11 @@ IMAP (OC_User_IMAP), SMB (OC_User_SMB), and FTP (OC_User_FTP).
   ],
 ....
 
-=== Define a custom link to reset password.
-
+=== Define a custom link to reset password
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc]: This issn't quite correct. I'd change it to: "Define a custom link for resetting passwords".

**Comment by phil-davis** [comment]
IMO these changes come from the changes merged in core yesterday.
Any problems with the text should have been reviewed there.
Another PR will be needed in core to adjust any of this.

File: modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc [hunk 3]
```diff
@@ -1030,7 +1008,7 @@ Supported conditions:
                 this logfile, this condition is met
                 Note: Not applicable when using syslog.
 
-Defaults to an empty array.
+Defaults to an empty array
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc]: Why was the full stop removed from the end of this sentence?

File: modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc [hunk 4]
```diff
@@ -2017,8 +1963,7 @@ using redis is highly recommended to *avoid data loss*.
 'upgrade.disable-web' => false,
 ....
 
-=== Defines the automatic update of market apps.
-
+=== Defines the automatic update of market apps
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc]: The sentence should start in the singular, as most others do.

**Comment by mmattel** [comment]
@settermjd this PR is a build based on changes of core, just as @phil-davis has written.
So either you do a PR that reflects your suggestions in core, or you approve/merge/backport this PR so this will be visible in docs and do a post round fixing your suggestions.

**Comment by mmattel** [comment]
@phil-davis why is CI stuck (not started at all) ??

**Comment by phil-davis** [comment]
> @phil-davis why is CI stuck (not started at all) ??

you are just lucky I guess :)

The drone job should at least be queued and have a link to it.

You will have to rebase or similar to get it to try and trigger drone again.

**Comment by settermjd** [comment]
Thanks for pointing out those considerations, @phil-davis @mmattel.

**Comment by mmattel** [comment]
> you are just lucky I guess :)

I am magnetic identifying possible improvements :smile:

**Comment by mmattel** [comment]
Just updated the PR based on the lastest PR in master

**Pull request merged**